indico/modules/events/surveys/templates/display/survey_questionnaire.html

```diff
@@ -49,7 +57,23 @@
     {% endcall %}
 
     <script>
-        // allow deselecting a horizontal radio button unless it's required
-        $('#survey-questionnaire .inline-vcentered input:radio:not([required])').resettableRadioButtons();
+        (function() {
+            'use strict';
+            // allow deselecting a horizontal radio button unless it's required
+            $('#survey-questionnaire .inline-vcentered input:radio:not([required])').resettableRadioButtons();
+            $('.js-save-answers').on('declarative:success', function() {
+                var $form = $(this).closest('form');
+                cornerMessage({
+                    actionLabel: $T.gettext('Submit'),
+                    progressMessage: $T.gettext('Submitting...'),
+                    message: $T.gettext('Your answers have been saved, do not forget to submit when you finish!'),
```

Comment:
I've tried exactly that same sentence before but then the corner message split in two lines and was uglier. I can change it though.